### PR TITLE
finishing setup leaderboard project

### PR DESCRIPTION
**Pull request by mohamed20163858**
setup the leaderboard project by installing webpack and writing minimal HTML,  CSS, and JavaScript  code to build a basic wireframe for the project

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.eslintrc.json
+.stylelintrc.json
+package.json
+package-lock.json
```
Comment (Munsa1) [inline .gitignore]: - [x] Kindly remove these files from the `.gitignore` file as they are essential to this project. the first two are linter config files, and the second two hold your project configurations which are part of the code review process. 
- [x] I don't know how but the package-lock.json file is on the branch but the rest are missing
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @mohamed20163858, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (mohamed20163858) [reply .gitignore]: @Munsa1  resolved thanks a lot for mentioning that

**Review by andersonlebon — APPROVED** [review]
## Hi @mohamed20163858 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**